pkgs/applications/science/electronics/kicad/default.nix

```diff
@@ -106,13 +111,42 @@ let
   wxGTK = wxGTK32;
   python = python3;
   wxPython = python.pkgs.wxPython_4_2;
+  thirdPartyPackagePath = "package.zip";
+  thirdPartyPackagesDrvs = map (pkg: pkg.override { inherit thirdPartyPackagePath python3; }) thirdPartyPackages;
+
+  thirdPartyPackagesJoined =
+    let
+      thirdParty = "$out/share/kicad/scripting";
+    in
+    runCommand "thirdPartyPackagesJoined"
+      {
+        inherit thirdPartyPackagesDrvs;
+      } ''
+      mkdir $out
+
+      for pkg in $thirdPartyPackagesDrvs; do
+        ${lib.getBin unzip}/bin/unzip $pkg/package.zip -d unpacked
```

Comment:
unzip should be added to nativeBuildInputs instead

Reply:
Marking resolved as we did this.